### DISCUSSION TITLE
Note in Developer Log when a tag cannot be processed?

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#4124] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
When a template calls a tag that doesn't exist, you get an error page that says "The following tag cannot be processed"

![tag_error_display](https://github.com/ExpressionEngine/ExpressionEngine/assets/49483/c022eae4-0bc4-492f-a73f-3faf30c14db4)

**Describe the solution you'd like**
It would be helpful if these errors were logged in the Developer Log, too.

**Describe alternatives you've considered**
Hmm. None? 

**Teachability, Documentation, Adoption, Migration Strategy**

- [ x ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
@nep thank you for the suggestion.

We are working on a better logging solution for ExpressionEngine and we'll make sure this case gets covered.
